ui/app/AppLayouts/Wallet/popups/swap/SwapModalAdaptor.qml

```diff
@@ -189,4 +223,37 @@ QObject {
         }
         return totalBalance
     }
+
+    function fetchSuggestedRoutes() {
+        // Identify new swap with a different uuid
+        d.uuid = Utils.uuid()
+
+        let account = getSelectedAccount(root.swapFormData.selectedAccountIndex)
+        let accountAddress = account.address
+        let disabledChainIds = getDisabledChainIds(root.swapFormData.selectedNetworkChainId)
+        let preferedChainIds = getAllChainIds()
+
+        root.swapStore.fetchSuggestedRoutes(accountAddress, accountAddress, 
+            root.swapFormData.fromTokenAmount, root.swapFormData.fromTokensKey, root.swapFormData.toTokenKey,
```

Comment:
this needs to be converted to 10^decimals?

Reply:
added TODO regarding changing this when the input component is integrated